### PR TITLE
chore: defer people to docs for getting started

### PR DESCRIPTION
**Pull request by james-elicx**
I don't think it makes sense to maintain two separate guides, especially when they both have a lot of overlap. We should just maintain the guide for starting/migrating apps in the docs and defer users to that page.

### CODE REVIEW

File: packages/cloudflare/README.md [hunk 1]
```diff
@@ -2,109 +2,8 @@
```
Comment (vicb) [inline packages/cloudflare/README.md:2]: I think we should keep the content here as it appears on https://www.npmjs.com/package/@opennextjs/cloudflare

What do other think?

**Review by dario-piotrowicz — APPROVED** [review]
I'd be happy to proceed with this PR's changes

@vicb please let me know what you think, if you strongly disagree we can think of something else and decline the PR, I'd just avoid to keep this lingering indefinitely

File: packages/cloudflare/README.md (hunk 1 above)
Comment (petebacondarwin) [reply packages/cloudflare/README.md:2]: I don't feel strongly but would also be happy to land this if @vicb is not strongly against it. Avoiding having to maintain two lots of docs is a good idea. That being said, do we have a plan (other than @dario-piotrowicz's suggestion above) to ensure the docs are kept up to date?

**Review by petebacondarwin — APPROVED** [review]

File: packages/cloudflare/README.md (hunk 1 above)
Comment (vicb) [reply packages/cloudflare/README.md:2]: Sure 🚀

**Pull request merged**